src/placeos-rest-api/controllers/root.cr

```diff
@@ -114,7 +114,8 @@ module PlaceOS::Api
     end
 
     private def self.core_version : (PlaceOS::Model::Version | Nil)
-      Core::Client.client(PLACE_URI, &.version)
+      uri = URI.new(host: CORE_HOST, port: CORE_PORT, scheme: "http")
+      Core::Client.client(uri, &.version)
```

Comment:
You want `core_for`

https://github.com/PlaceOS/rest-api/blob/156d88a1553fe798ca867f1350896e809b0650b6/src/placeos-rest-api/controllers/systems.cr#L404-L407

this lines i sent could also be moved into `Application` as it's used around the place.